src/cmd_line/commands/tab.ts

```diff
@@ -119,17 +119,17 @@ export class TabCommand extends node.CommandBase {
         await vscode.commands.executeCommand('workbench.action.closeOtherEditors');
         break;
       case Tab.Move:
-        if (this._arguments.count !== undefined) {
-          if (this._arguments.count === 0) {
-            vscode.commands.executeCommand('activeEditorMove', { to: 'first' });
-          } else {
-            vscode.commands.executeCommand('activeEditorMove', {
-              to: 'position',
-              amount: this._arguments.count,
-            });
-          }
+        // Documentation: https://github.com/Microsoft/vscode/issues/8234#issuecomment-234573410
```

Comment:
Can you remove the comment? I'm not a fan of having such comments in the code.

Reply:
Comment removed. I put the link on this issue instead.